### DISCUSSION TITLE
Metamorph

### DISCUSSION THREAD

**Discussion by kekeishere** [Crownicles/Crownicles#2095] [Suggestions fermées]
On pourrais ajouter un 5 metamorph avec 5 raretés differentes allant de 1⭐ à 5⭐. Par exemple il y aurait un metamorph 1⭐, qui s'obtient comme tous les pets 1 etoiles mais sa probabilité d'obtention est celle d'un 2⭐ mais les recompenses restent celle d'un 1⭐ sauf si in est dressé alors c'est les recompenses d'un 2⭐. 
En obtenant un metamorph 1⭐ on a la possibilité de choisir en quel animal il va se métamorphoser parmis tous les pets 1⭐, ou sinon on le laisse dans l'état dans lequel il a trouvé, sa donne ça : 
Type : 🕳 Metamorph
Surnom : Aucun surnom

Rareté : ⭐
Sexe : Femelle ♀️ 
Mentalité : 😾 Sauvage

On peut choisir de changer la forme du metamorph via une commande /transform. En principe les metamorph ne gardent pas le transformation définitivement, dans ce cas ils pourront reprendre leur forme d'origine à intervalles de 1mois ou bien 2 semaines toujours via la commande /transform. 
Mais pour le transformer il faut qu'il soit au minimum apprivoisé, je pense qu'il serait cool de faire en sorte qu'après chaque transformations le metamorph perde le l'amitié genre 50 ou bien 25.

◾ Metamorph ⭐ c'est comme si on pack un 2⭐
⬛ Metamorph ⭐⭐ c'est comme si on pack un 3⭐
⚫ Metamorph ⭐⭐⭐ c'est comme si on pack un 4⭐
🕳 Metamorph ⭐⭐⭐⭐ c'est comme si on pack un 5⭐
✴ Metamorph ⭐⭐⭐⭐⭐ probabilité d'obtention un peu plus dure 

Étant donné que le bot peu ajouter des emoji personnalisés si ces emoji ne vous plaisent pas je peux vous créer des metamorph que je dessinerai moi même un peu comme ma photo de profil sur discord, biensur je m'appliquerai et j'enleverai l'arriere plan de l'image.

Franchement je trouve que le concept des metamorphs est insane dans le style médiéval on pourrait retrouver des creatures mythologiques du style, sa collerai bien avec le style du jeu. 

N'hésitez si y'a des question :]

**Comment by Nargacaura** [comment]
C'est pas le même concept que la #1795, ça?

**Reply by kekeishere** [reply]
Non, mon ancien post etait pas assez restreint au niveau réglementation on était trop libre de faire n importe quoi, la c'est plus limité tu vois na difference ?

**Comment by kekeishere** [comment]
dcp vous en pensez quoi ?

**Comment by BastLast** [comment]
Compliqué à coder et imo peu d'intérêt de gameplay, je ne vois pas un soutient énorme de la communauté donc je deplace les 2 suggestions dans refusé

Aussi on peut pas juste copypast un personnage de jeu vidéo existant comme ça

**Reply by kekeishere** [reply]
c'est pas du contraste le personnage est different
